lib/msal-react/src/hooks/useAccount.ts

```diff
@@ -42,7 +42,10 @@ export function useAccount(accountIdentifiers: AccountIdentifiers): AccountInfo
     const [account, setAccount] = useState<AccountInfo|null>(initialStateValue);
 
     useEffect(() => {
-        setAccount(getAccount(instance, accountIdentifiers));
+        const currentAccount = getAccount(instance, accountIdentifiers);
+        if (!AccountEntity.accountInfoIsEqual(account, currentAccount, true)) {
+            setAccount(currentAccount);
+        }
     }, [inProgress, accountIdentifiers, instance]);
```

Comment:
Thoughts on the warning below?

Reply:
Thanks, addressed. May consider making that an error instead of a warning